content/projects/blueocean/index.html.haml

```diff
@@ -21,11 +21,13 @@ links:
           .logo
             = partial('logo.html.haml')
           %h1.intro-title
-            Jenkins Blue Ocean
+            Blue Ocean for Jenkins Pipelines
           %a.btn.btn-secondary{:href => expand_link('doc/book/blueocean')}
             Documentation
-          %a.btn.btn-secondary{:href => expand_link('projects/blueocean/roadmap')}
-            Roadmap
+          %a.btn.btn-secondary{:href => expand_link('projects/blueocean/releases')}
```

Comment:
Since the development team has now stated that they are publishing release notes on GitHub, we should link to those release notes rather than linking to a page that does not exist in the site.
```suggestion
          %a.btn.btn-secondary{:href => 'https://github.com/jenkinsci/blueocean-plugin/releases'}
```